source/ranges.tex

```diff
@@ -2399,6 +2402,10 @@
 
 \begin{itemdescr}
 \pnum
+\mandates
+For the first overload,
+\tcode{C} is a cv-unqualified class type.
```

Comment:
```suggestion
\tcode{C} is a cv-unqualified class type.

```
A `\pnum` follows. If I suggest on that line, when applied, this comment would be "outdated" rather than "resolved".